export const mockErrorUserAlreadyExists = {
  statusCode: 422,
  error: "User already exists.",
};

export const mockErrorInternalServer = {
  statusCode: 500,
  error: "Internal Server Error",
};
